src/hooks/player_management/assign_physical_index.cpp

```diff
@@ -115,16 +115,7 @@ namespace big
 					}
 
 					if (plyr->block_join && *g_pointers->m_gta.m_is_session_started)
-					{
-						if (g_player_service->get_self()->is_host())
-						{
-							dynamic_cast<player_command*>(command::get("breakup"_J))->call(plyr, {});
-						}
-						else if (g.protections.desync_kick)
-						{
-							dynamic_cast<player_command*>(command::get("desync"_J))->call(plyr, {});
-						}
-					}
+						dynamic_cast<player_command*>(command::get("smartkick"_J))->call(plyr, {});
```

Comment:
Is this indented properly?

Reply:
yes

Reply:
Oh yeah sorry, I was looking on github mobile. this can be resolved.